CFP.md

```diff
@@ -0,0 +1,63 @@
+CALL FOR PAPERS
+====
+
+--------------------------------------------------------------------------------
+*Workshop for NLP Open Source Software (NLP-OSS)*
+20 July 2018, Co-located with ACL 2018
+https://nlposs.github.io/
+
+Deadline for Long and Short Paper submission: 25th March 2018 
+(23:59, GMT-11)
+---------------------------------------------------------------------------------
+
+The First Workshop for NLP Open Source Software (NLP-OSS) will be co-located 
+with ACL 2018 at Melbourne, Australia on 20 July 2018. 
+
+The Workshop for NLP-OSS hopes to be the intellectual forum to collate engineering 
+knowledge focusing more on the building aspect of NLP and less on scientific 
+novelty or state-of-art development. 
+
+NLP-OSS also provides an the academic workshop platforms to announce new software/features, 
+promote the open source culture and best practices that go beyond the conferences.
+
+We invite full papers (8 pages) or short papers (4 pages) on topics related to NLP-OSS 
```

Comment:
Ordinarily a CFP specifies a genre of paper. I think that's what you are trying to do here, particularly with "case studies". By scientific contribution, I'm not sure if you intend "empirical analysis". And your "software development" examples are might be a mixed bag of case studies, critical reflection, surveys, proposals (white papers), etc. I suppose what I am saying is that this categorisation confuses theme and genre, and that I think it would be helpful to describe the sought genres, to help contributors structure their work.